distributed/semaphore.py

```diff
@@ -204,6 +204,24 @@ def close(self, comm=None, name=None):
 class Semaphore:
     """ Semaphore
 
+    This semaphore will track leases on the scheduler which can be acquired and
+    released by an instance of this class. If the maximum amount of leases are
+    already acquired, it is not possible to acquire more and the caller waits
+    until another lease has been released.
+
+    The lifetime of a lease are coupled to the ``Client`` it was acquired with.
+    Once the Client goes out of scope, the leases associated to it are freed.
+    This behaviour can be controlled with the parameter
```

Comment:
```suggestion
    This behavior can be controlled with the parameter
```
both are valid spellings but the rest of the dask docs use the American version